src/compiler/scala/tools/nsc/backend/jvm/BTypes.scala

```diff
@@ -625,7 +625,7 @@ abstract class BTypes {
         // synchronization required to ensure the apply is finished
         // which populates info. ClassBType doesnt escape apart from via the map
         // and the object mutex is locked prior to insertion. See apply
-        this.synchronized()
+        this.synchronized { () }
```

Comment:
`this synchronized ()` isn't good enough but `this synchronized {}` gets you parens for free.

Reply:
Sorry I was a bit parenoid there.